src/main/resources/com/unstoppabledomains/config/knownRecords.json

```diff
@@ -2625,6 +2625,16 @@
       "deprecatedKeyName": "GTH",
       "validationRegex": "^0x[a-fA-F0-9]{40}$",
       "deprecated": false
+    },
```

Comment:
Did you update to the latest UNS and resolver config? If so, please update the PR description to keep track of the change.